packages/storybook/stories/Overview.stories.mdx

```diff
@@ -64,28 +64,128 @@ npm i --save @azure/communication-react
+
+### Main Components for Calling
+
+<Stack style={{ backgroundColor: '#EFF6FC', padding: '2.25rem' }}>
+  <Stack.Item align="center">
+    <img src="images/components_calling.png" alt="example of components that can be used for calling" />
+  </Stack.Item>
+</Stack>
+
+| Component                                                                 | Description                                                                                        |
+| ------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------- |
+| [Grid Layout](./?path=/story/ui-components-gridlayout--grid-layout)       | Grid component to organize Video Tiles into an NxN grid                                            |
+| [Video Tile](./?path=/story/ui-components-videotile--video-tile)          | Component that displays video stream when available and a default static component when not        |
+| [Control Bar](./?path=/story/ui-components-controlbar--control-bar)       | Container to organize DefaultButtons to hook up to specific call actions like mute or share screen |
+| [Video Gallery](./?path=/story/ui-components-videogallery--video-gallery) | Turn-key video gallery component which dynamically changes as participants are added               |
+
+### Main Components for Chat
+
+<Stack style={{ backgroundColor: '#EFF6FC', padding: '2.25rem' }}>
+  <Stack.Item align="center">
+    <img src="images/components_chat.png" alt="example of components that can be used for chat" />
+  </Stack.Item>
+</Stack>
+
+| Component                                                                                                 | Description                                                                   |
+| --------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------- |
+| [Message Thread](./?path=/story/ui-components-messagethread--message-thread)                              | Container that renders chat messages, system messages, and custom messages    |
+| [Send Box](./?path=/story/ui-components-sendbox--send-box)                                                | Text input component with a discrete send button                              |
+| [Message Status Indicator](./?path=/story/ui-components-messagestatusindicator--message-status-indicator) | Multi-state message status indicator component to show status of sent message |
+| [Typing indicator](./?path=/story/ui-components-typingindicator--typing-indicator)                        | Text component to render the participants who are actively typing on a thread |
+
+### Common Components
+
+<Stack style={{ backgroundColor: '#EFF6FC', padding: '2.25rem' }}>
+  <Stack.Item align="center">
+    <img src="images/components_common.png" alt="example of components that can be used for chat" />
```

Comment:
As participant pane is in beta I think we should avoid references on the homepage to participant pane being in chat